targets/targets.json

```diff
@@ -2655,11 +2678,17 @@
             "STM32F429ZI",
             "STM32F429xx",
             "STM32F429xI",
-            "STM_EMAC"
+            "STM_EMAC",
+            "PSA"
         ],
+        "components_add": ["FLASHIAP"],
         "macros_add": [
             "USB_STM_HAL",
-            "USBHOST_OTHER"
+            "USBHOST_OTHER",
+            "MBEDTLS_PSA_CRYPTO_C",
+            "MBEDTLS_ENTROPY_NV_SEED",
```

Comment:
This device has TRNG enabling nvseed is redundant

Reply:
Pelion require injection API regardless TRNG